pkg/config/orderer.go

```diff
@@ -19,38 +19,49 @@ import (
 	eb "github.com/hyperledger/fabric-protos-go/orderer/etcdraft"
 )
 
-// Orderer encodes the orderer-level configuration needed in config transactions.
+// Orderer is the orderer implementation details.
```

Comment:
It doesn't hold implementation details. It may configure the ordering service behavior for the channel...